src/css/components/_layout.scss

```diff
@@ -124,6 +124,10 @@ body.vjs-full-window {
 // Hide disabled or unsupported controls.
 .vjs-hidden { display: none !important; }
 
+.vjs-disabled {
```

Comment:
we can also change the [`cursor` `not-allowed`](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#Values)